interp/constrexpr_ops.ml

```diff
@@ -154,6 +154,7 @@ let rec constr_expr_eq e1 e2 =
     Id.equal id1 id2 && List.equal instance_eq c1 c2
   | CSort(_,s1), CSort(_,s2) ->
     Miscops.glob_sort_eq s1 s2
+  (* Why do we distinguish kinds of casts? *)
   | CCast(_,a1,(CastConv b1|CastVM b1)), CCast(_,a2,(CastConv b2|CastVM b2)) ->
```

Comment:
@herbelin Indeed, not factorizing different casts looks simpler and closer to the expected behavior. I'll test that as a separate patch.

About `:>`, I have no idea. I didn't know it existed before reading your message.